public/app/features/manage-dashboards/state/actions.ts

```diff
@@ -222,6 +223,14 @@ export function createFolder(payload: any) {
   return getBackendSrv().post('/api/folders', payload);
 }
 
+export function searchFolders(query: any, permission?: 'View' | 'Edit'): Promise<DashboardSearchHit[]> {
```

Comment:
Nit: `'View' | 'Edit'` - can we make it an enum?

Reply:
Good suggestion, I'll fix it!

Reply:
Fixed in https://github.com/grafana/grafana/pull/38103/commits/a96a50ed929511dcdbfcd2e91e0cfdb027a4e4c2